fuzzers/centipede/runner.Dockerfile

```diff
@@ -13,3 +13,5 @@
 # limitations under the License.
 
 FROM gcr.io/fuzzbench/base-image
+
```

Comment:
Youre not doing this correctly. Need to add a from ... As base-clang.
See https://github.com/google/fuzzbench/blob/d35a6951e2f05b7bab34f9a8c94e381c9512f903/fuzzers/libfuzzer_focus_idx0/runner.Dockerfile#L15

Reply:
Done, thanks!